borg/cache.py

```diff
@@ -311,7 +311,7 @@ def create_master_idx(chunk_idx):
             cleanup_outdated(cached_ids - archive_ids)
             if archive_ids:
                 chunk_idx = None
-                for archive_id in archive_ids:
+                for archive_id in sorted(archive_ids):
```

Comment:
I'll remove that commit in squashing, it was a testing-only thing (no harm in it, just off-topic), to make the chunks cache creation deterministic. This made it much easier to see if there where any changes in the generated files.